raiden-dapp/src/views/Home.vue

```diff
@@ -86,18 +87,23 @@ export default class Home extends Vue {
     );
   }
 
-  async connect(subkey?: true) {
+  async connect(connectOptions: ConnectOptions) {
+    const stateBackup = connectOptions['uploadedState'] || undefined;
```

Comment:
is there any specific reason for `connectOptions['uploadedState']` instead of `connectOptions.uploadedState`

Reply:
Nope, the only reason was just me not knowing better. :)